tests/core_tests.py

```diff
@@ -157,6 +172,39 @@ def test_slices(self):
             logging.info("[{name}]/[{method}]: {url}".format(**locals()))
             self.client.get(url)
 
+    def test_add_slice(self):
+        self.login(username='admin')
+
+        # Click on the + to add a slice
+        url = '/slicemodelview/add'
+        resp = self.get_resp(url)
+        assert 'Click on a' in resp
+        assert 'to create a Slice' in resp
+
+        tables = db.session.query(models.SqlaTable).all()
+        for table in tables:
+            # Check that the /tablemodelview/list view contains each table and
+            # their show, edit, delete buttons
+            assert table.name in resp
+            assert '/superset/explore/table/{}'.format(table.id) in resp
+            assert '/tablemodelview/show/{}'.format(table.id) in resp
+            assert '/tablemodelview/edit/{}'.format(table.id) in resp
+            assert '/tablemodelview/delete/{}'.format(table.id) in resp
+
+            # Visit each table's explore view and make sure some
```

Comment:
Do you have any advice for what tests I should include here? Should I not test whether each table name appears along with the edit/show/delete buttons?